salt/recommendations/init.sls

```diff
@@ -57,24 +49,52 @@ recommendations-configuration:
         - group: {{ pillar.elife.deploy_user.username }}
         - template: jinja
         - require:
-            - recommendations-repository
+            - recommendations-folder
+
+recommendations-docker-compose-env:
+    file.managed:
+        - name: /srv/recommendations/.env
+        - source: salt://recommendations/config/srv-recommendations-.env
+        - user: {{ pillar.elife.deploy_user.username }}
+        - group: {{ pillar.elife.deploy_user.username }}
+        - template: jinja
+        - require:
+            - recommendations-folder
+
+recommendations-docker-compose-containers-env:
+    file.managed:
+        - name: /srv/recommendations/containers.env
+        - source: salt://recommendations/config/srv-recommendations-containers.env
+        - user: {{ pillar.elife.deploy_user.username }}
+        - group: {{ pillar.elife.deploy_user.username }}
+        - template: jinja
+        - require:
+            - recommendations-folder
 
-recommendations-composer-install:
+# deprecated, remove when no longer necessary
+# if not stopped, may conflict with port 9000 forwarded from the host to the container
```

Comment:
Probably want a `require_in`?

Reply:
And follow-up removing PHP.

Reply:
Can add it, the order of states in the file should be respected but it doesn't hurt.

Reply:
Relying on that works, until it doesn't.

Reply:
Adding requirements but removal of PHP and other strange files will be done by substituting the whole server instead when migrating to 16.04.